docs/enterprise-app-store/index.md

```diff
@@ -110,69 +112,60 @@ A download is calculated every time an app is downloaded from our servers. So a
-#in the --app "$AC_OUTPUT_DIR"/*.apk code line down below
+# If an IPA or AAB is required, change *.apk to *.ipa or *.aab
 appcircle enterprise-app-store version upload-for-profile \
   --entProfileId "$ORG_B_ENT_APP_STORE_PROFILE_ID" \
   --app "$AC_OUTPUT_DIR"/*.apk
 ```
 
-#### Upload binary with non-existing Enterprise App Store profile
+#### Uploads a binary and creates the Enterprise App Store profile if it does not already exist
 
 ```bash
 #Bash script
 sudo npm install -g @appcircle/cli
 appcircle login --pat $ORG_B_PERSONAL_API_TOKEN
-#if ipa or aab is required change it to *.ipa or *.aab 
-#in the --app "$AC_OUTPUT_DIR"/*.apk code line down below
+# If an IPA or AAB is required, change *.apk to *.ipa or *.aab
 appcircle enterprise-app-store version upload-without-profile \
   --app "$AC_OUTPUT_DIR"/*.apk
 ```
 
 This will also generate a new Enterprise App Store profile and application will be sent into this profile.
 
-:::caution
-
-Keep in mind that uploading a binary to existing Enterprise App Store profile will require newer version code of the Application.
-
-:::
+<NewerVersionCodeCaution />
 
 The key point here is that we need two essential parameters to make this work.
 - `ORG_B_PERSONAL_API_TOKEN` => Organization PAT (Personal API Token) from Organization B 
 - `ORG_B_ENT_APP_STORE_PROFILE_ID` => Enterprise App Store profile ID from Organization B
 
-To generate Personal API Token, follow this documentation [API authentication](/appcircle-api-and-cli/api-authentication/)
-
-You can get Enterprise App Store profile ID by logging into Organization B, selecting the desired Enterprise App Store profile, and copying it from the URL.
-For example, let's assume this is the URL after selecting the Enterprise App Store Profile.
-`https://my.appcircle.io/enterprise-store/profiles/123456f-7d89-4545-5454-123456789abc`
+`$AC_OUTPUT_DIR` is an automatically generated environment variable. [Reserved Variables](/environment-variables/appcircle-specific-environment-variables/)
 
-Then the Enterprise App Store profile ID is => `123456f-7d89-4545-5454-123456789abc`
+To generate Personal API Token, follow this documentation [API authentication](/appcircle-api-and-cli/api-authentication/)
 
-`$AC_OUTPUT_DIR` is an automatically generated environment variable. [Reserved Variables](/environment-variables/appcircle-specific-environment-variables/)
+To obtain the Enterprise App Store profile ID, follow the steps below: 
+1. Log in to organization B.
+2. Go to Enterprise App Store module.
+3. Select the desired Enterprise App Store profile
+4. Copy it from the URL. `https://my.appcircle.io/enterprise-store/profiles/123456f-7d89-4545-5454-123456789abc`
+5. Then the Publish profile ID is => `123456f-7d89-4545-5454-123456789abc`
 
 After collecting the essential parameters, they have to be set in the [Environment Variables](/environment-variables/) as 
 ORG_B_PERSONAL_API_TOKEN,
 ORG_B_ENT_APP_STORE_PROFILE_ID
```

Comment:
Could you wrap these variables in backticks (``)?
Also, I noticed that you wrote the environment variables on new lines, but they appear on the same line in the preview. If you want them to appear on separate lines, you should leave an empty line above each one.